guides/source/active_storage_overview.md

```diff
@@ -548,6 +548,57 @@ jobs, Cronjobs, etc.), you can access the `rails_blob_path` like this:
 Rails.application.routes.url_helpers.rails_blob_path(user.avatar, only_path: true)
 ```
 
+### Proxy mode
+
+Optionally, files can be proxied instead. This means that your application servers will download file data from the storage service in response to requests. This can be useful for serving files from a CDN.
+
+Explicitly proxy attachments using the `rails_storage_proxy_path` and `_url` route helpers:
```

Comment:
```suggestion
If you want to explicitly proxy specific attachments there are URL helpers you can use in the form of `rails_storage_proxy_path` and `rails_storage_proxy_url`.
```

Reply:
this was copy pasted from the ActiveStorage README, but I can update it in both places if you'd like

Reply:
@santib Sounds good, let's update both while we're here